pandapower/test/api/test_toolbox.py

```diff
@@ -165,6 +166,21 @@ def test_nets_equal():
     assert tb.nets_equal(original, net, atol=0.1)
     assert tb.nets_equal(net, original, atol=0.1)
 
+    # check controllers
+    original.trafo.tap_side.fillna("hv", inplace=True)
+    net1 = original.deepcopy()
+    net2 = original.deepcopy()
+    pp.control.ContinuousTapControl(net1, 0, 1.0)
+    pp.control.ContinuousTapControl(net2, 0, 1.0)
+    c1 = net1.controller.object.at[0]
+    c2 = net2.controller.object[0]
+    assert c1 == c2
+    assert tb.nets_equal(net1, net2)
+    c1.vm_set_pu = 1.01
+    assert c1 != c2
+    assert tb.nets_equal(net1, net2, exclude_elms=["controller"])
```

Comment:
Shall we add that nets_equal is comparing the attributes but not the adress in the function nets_equal?

Reply:
added in the docstring